splash/lua_modules/wraputils.lua

```diff
@@ -169,6 +170,9 @@ end
 -- Create a Lua wrapper for a Python object.
 --
 local function wrap_exposed_object(py_object, self, cls, private_self, async)
+  setmetatable(self, cls)
+  cls.__newindex = rawset
```

Comment:
hmm, does this metamethod only live up to `setup_property_access` where it's reset with setter-related one?